src/main/java/com/mentalhealthapi/mentalhealthapi/service/IDisorderService.java

```diff
@@ -7,7 +7,6 @@
 
 @Service
 public interface IDisorderService {
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure